### ISSUE TITLE
And on the "clone site" it is playing files from my AWS_S3 CDN setup, but does not have a AWS_S3 plugin??? Installing AWS_S3 plug-in killed the site.

### ISSUE THREAD

**Issue by walt93** [WWBN/AVideo#8376]
**Describe the bug**
Took a git pull. It had been a while, there were 5 database updates. Now site is completely down. No /video page works; video is not displayed. Git pull taken Sun Sep 3. 

**Screenshots**
All videos render like this:
![Screen Shot 2023-09-03 at 3 25 58 PM](https://github.com/WWBN/AVideo/assets/11711050/3f413564-66a1-469b-80e2-78bdf095ec42)

**Logs**
I don't see anything.  What do I even look for?

```
[03-Sep-2023 15:23:09 America/Los_Angeles] AVideoLog::DEBUG: Commit transaction https://burrow.conspyre.net/view/ SCRIPT_NAME: /view/index.php
[03-Sep-2023 15:23:12 America/Los_Angeles] AVideoLog::DEBUG: NOT isShareEnabled() SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:12 America/Los_Angeles] AVideoLog::DEBUG: CustomizeUser::canShareVideosFromVideo canShareVideosFromUser(1) = false SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:12 America/Los_Angeles] AVideoLog::DEBUG: NOT isShareEnabled() SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:12 America/Los_Angeles] AVideoLog::DEBUG: CustomizeUser::canShareVideosFromVideo canShareVideosFromUser(1) = false SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:12 America/Los_Angeles] AVideoLog::DEBUG: NOT isShareEnabled() SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:12 America/Los_Angeles] AVideoLog::DEBUG: CustomizeUser::canShareVideosFromVideo canShareVideosFromUser(1) = false SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:12 America/Los_Angeles] AVideoLog::DEBUG: NOT isShareEnabled() SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:12 America/Los_Angeles] AVideoLog::DEBUG: CustomizeUser::canShareVideosFromVideo canShareVideosFromUser(1) = false SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:12 America/Los_Angeles] AVideoLog::DEBUG: Begin transaction https://burrow.conspyre.net/view/modeYoutube.php?v=5744 SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:12 America/Los_Angeles] AVideoLog::DEBUG: Commit transaction https://burrow.conspyre.net/view/modeYoutube.php?v=5744 SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:27 America/Los_Angeles] AVideoLog::DEBUG: Warning: Slow process detected [navbar.php] On  Line 120 takes 0.2298 seconds to complete, Limit (0.2). /var/www/html/burrow.conspyre.net/view/index.php SCRIPT_NAME: /view/index.php
[03-Sep-2023 15:23:27 America/Los_Angeles] AVideoLog::DEBUG: Begin transaction https://burrow.conspyre.net/view/?catName=great-reset&page=2&lang=pt SCRIPT_NAME: /view/index.php
[03-Sep-2023 15:23:27 America/Los_Angeles] AVideoLog::DEBUG: Commit transaction https://burrow.conspyre.net/view/?catName=great-reset&page=2&lang=pt SCRIPT_NAME: /view/index.php
[03-Sep-2023 15:23:34 America/Los_Angeles] AVideoLog::DEBUG: NOT isShareEnabled() SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:34 America/Los_Angeles] AVideoLog::DEBUG: CustomizeUser::canShareVideosFromVideo canShareVideosFromUser(1) = false SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:35 America/Los_Angeles] AVideoLog::DEBUG: NOT isShareEnabled() SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:35 America/Los_Angeles] AVideoLog::DEBUG: CustomizeUser::canShareVideosFromVideo canShareVideosFromUser(1) = false SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:35 America/Los_Angeles] AVideoLog::DEBUG: NOT isShareEnabled() SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:35 America/Los_Angeles] AVideoLog::DEBUG: CustomizeUser::canShareVideosFromVideo canShareVideosFromUser(1) = false SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:35 America/Los_Angeles] AVideoLog::DEBUG: NOT isShareEnabled() SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:35 America/Los_Angeles] AVideoLog::DEBUG: CustomizeUser::canShareVideosFromVideo canShareVideosFromUser(1) = false SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:35 America/Los_Angeles] AVideoLog::DEBUG: Begin transaction https://burrow.conspyre.net/view/modeYoutube.php?v=5768 SCRIPT_NAME: /view/modeYoutube.php
[03-Sep-2023 15:23:35 America/Los_Angeles] AVideoLog::DEBUG: Commit transaction https://burrow.conspyre.net/view/modeYoutube.php?v=5768 SCRIPT_NAME: /view/modeYoutube.php
```

**Comment by DanielnetoDotCom** [comment]
Check for plugins updates

the logs you shared does not help

**Comment by walt93** [comment]
Only two plugins needed updates: Customize and UserControll - both of which I have turned off

I updated Customize by dragging it into the [+Upload a Plugin] interface, and it worked.

I updated UserControll the same way, and it did not update. 

![Screen Shot 2023-09-03 at 3 48 40 PM](https://github.com/WWBN/AVideo/assets/11711050/e1c82954-6a21-4153-a820-523e8a26b68b)

Most of the plugins, I have not installed tables for on this site.

**Comment by walt93** [comment]
I now realize I didn't save my S3 settings and have to reconstruct them.

It would be dope if you fixed this so it doesn't continue blowing out peoples settings.

**Comment by DanielnetoDotCom** [comment]
https://github.com/WWBN/AVideo/wiki/How-To-Install-a-new-Plugin#addressing-permission-issues

also install all the tables

**Comment by DanielnetoDotCom** [comment]
Also it should not clear your settings.

**Comment by walt93** [comment]
It did, unfortunately. I noticed that AWS_S3 was off. I went to check it, and it comes up blank.

It also needed me to update its table.

![Screen Shot 2023-09-03 at 3 56 45 PM](https://github.com/WWBN/AVideo/assets/11711050/57bbf2e8-4ec4-40d9-83d7-d3d337818931)

**Comment by DanielnetoDotCom** [comment]
No idea why it is clear. 

this is not supposed to happen. 

the only solution now it to get it again from your S3 provider

**Comment by walt93** [comment]
Yeah that's what I'm doing.

Are you saying I should install all the tables, even for the ones I don't use? User Location is like huge, isn't it?

**Comment by DanielnetoDotCom** [comment]
yes, install them

**Comment by walt93** [comment]
Daniel, I'm stumped. I have a clone of the site, it's running. All I should have to do is copy the settings. How is this possible? 

The backup site is apparently missing the AWS_S3 plugin. It's not there. But I don't have any storage to host the files, the site works, and if I inspect the videos page - the files are coming from my CDN.

**Comment by walt93** [comment]
Ok, on the "clone site", which is functional, plays videos from the CDN, and apparently - does not have the AWS_S3 plugin installed??? (I know it MUST have one installed, otherwise how can it be playing files from the CDN???)

I went to update its plugins, and when I went to add the AWS_S3 plug-in - I could not - it crashes in the plug-ins install UI.
![Screen Shot 2023-09-03 at 4 17 53 PM](https://github.com/WWBN/AVideo/assets/11711050/6f224cd8-15c2-446c-839e-43bdb4555508)

**Comment by walt93** [comment]
I am so confused.

**Comment by DanielnetoDotCom** [comment]
you are mixing versions. 

if you made a git pull your plugins also must be the latest. download them again

**Comment by walt93** [comment]
I seem not to understand how this site is configured. I have it pulling files from the CDN, yet, it is doing so without using the AWS_S3 plugin.

I have essentially two copies of the same site. One is working. One died after I did a git pull. I'm not sure how to copy the configuration, since it doesn't make sense.

Can you make any sense of this Daniel?

**Comment by DanielnetoDotCom** [comment]
Sorry, I am very confused about what you are facing. but I am pretty sure you are mixing versions.

**Comment by walt93** [comment]
![Screen Shot 2023-09-03 at 4 28 35 PM](https://github.com/WWBN/AVideo/assets/11711050/5e913554-7561-4dc1-8f69-70a3ac32010b)

This is from the /video page (https://burrow.conspyre.tv/v/5668) - look at the URL - it is coming from "burrow2.b-cdn.net" - I know that when I set this site up, I did so using the AWS_S3 plugin.

But I'm looking at the AWS_S3 plugin and the configuration is blank. It actually wasn't even installed, I just had to install it. So how is the site pulling (correctly) from my CDN? Am I not remembering where the CDN config is?

It is, right here, on the AWS_S3 plug-in, which is not configured. But the site is working. How???

![Screen Shot 2023-09-03 at 4 30 34 PM](https://github.com/WWBN/AVideo/assets/11711050/a9e2c6a2-0bdb-42df-a5c7-36b4f5cec29d)

**Comment by DanielnetoDotCom** [comment]
If you bucket is public and the CDN plugin is pointing correctly, it should be fine

but you probably will not be able to submit new videos until you set it up again.

**Comment by walt93** [comment]
Where is it configured to use my CDN, though?

**Comment by DanielnetoDotCom** [comment]
check the CDN plugin

**Comment by DanielnetoDotCom** [comment]
acctually  I can see you are usign bunny, check your customizeAdvanced

**Comment by walt93** [comment]
CDN plugin is off. I'm looking through customizeAdvanced - and - videosCDN is also blank???

How can this site POSSIBLY be working? 🤣

**Comment by DanielnetoDotCom** [comment]
cache maybe, otherwise no idea

**Comment by walt93** [comment]
Has to be. Ok so customize->advanced->videosCDN is for reading/playback and AWS_S3 is for writing/storing?

**Comment by walt93** [comment]
Just saying Daniel, both of these sites are clones, and after checking out recently my AWS_S3 and videosCDN settings are now apparently empty. Maybe see if there is a bug in the migration code. I know what I need to fix now...

**Comment by DanielnetoDotCom** [comment]
Anyway. if ou still have the old site you may be able to manually retrieve the S3 configuration.

**Comment by walt93** [comment]
Two questions: 

1) How would I manually find the S3 config? I'm realizing now how messy my config is between these sites… 🤦‍♂️

2) How safe am I to restore my config using the "Backup" plugin? Backup was from like 12.4 or some older version.

**Comment by DanielnetoDotCom** [comment]
> How would I manually find the S3 config? I'm realizing now how messy my config is between these sites… 🤦‍♂️

the easiest is if the old site still running and you can check the plugin
the harder option is to check into the database directly

> How safe am I to restore my config using the "Backup" plugin? Backup was from like 12.4 or some older version.

it should make no difference. 
but check if there was any error when you restored it.

**Comment by walt93** [comment]
I went into the site and set up the AWS S3. It didn't make any difference. I added the CDN URL to customize advanced, and now the site doesn't load. Where do I clear the CDN URL from customize advanced? I know my way around mysql, what table and field and such am I looking for?

After adding the plugins->customize advanced->cdn URL I get this, it totally broke the HTML:

![Screen Shot 2023-09-03 at 7 05 40 PM](https://github.com/WWBN/AVideo/assets/11711050/5f3e6534-f735-4260-9e0c-6ee1a6109431)
